balloc/balloc.c

```diff
@@ -2978,30 +3008,38 @@ static const struct m0_ad_balloc_ops balloc_ops = {
 	.bo_reserve_extent = balloc_reserve_extent,
 };
 
-static int balloc_trees_create(struct m0_balloc    *bal,
-			       struct m0_be_tx     *tx,
-			       const struct m0_fid *fid)
+static int balloc_trees_create(struct m0_balloc		*bal,
+				struct m0_be_tx		*tx,
+				const struct m0_fid	*fid,
+				m0_bcount_t		 indexcount)
 {
 	int rc;
+	int i;
+	int k;
 
-	rc = M0_BE_OP_SYNC_RET(op,
-		       m0_be_btree_create(&bal->cb_db_group_extents, tx, &op,
+	for (i = 0; i < indexcount; i++) {
+		rc = M0_BE_OP_SYNC_RET(op,
+		       m0_be_btree_create(&bal->cb_db_group_extents[i], tx, &op,
 				  &M0_FID_TINIT('b',
 						M0_BBT_BALLOC_GROUP_EXTENTS,
 						fid->f_key)),
-		       bo_u.u_btree.t_rc);
-	if (rc != 0)
-		return M0_ERR(rc);
+						bo_u.u_btree.t_rc);
```

Comment:
The indentation here is not right.
```
3020         for (i = 0; i < indexcount; i++) {
3021                 rc = M0_BE_OP_SYNC_RET(op,
3022                        m0_be_btree_create(&bal->cb_db_group_extents[i], tx, &op,
3023                                           &M0_FID_TINIT('b',
3024                                           M0_BBT_BALLOC_GROUP_EXTENTS,
3025                                           fid->f_key)),
3026                                        bo_u.u_btree.t_rc);
3027                 if (rc != 0)
3028                         goto out;
3029 
3030         }
```
The arguments should be aligned to the function call.
Arguments of a function, should be aligned together.

Reply:
Changes has been done. please have al ook.